src/app/home/home.component.html

```diff
@@ -36,13 +36,13 @@ <h1 class="page-header">EnergyPlus</h1>
         the results of two or more simulations. Several comprehensive
         <a href="https://www.ibpsa.us/best-directory-list/">
           tools</a> for EnergyPlus are also available. DOE does most of its work with EnergyPlus using the
-        <a href="https://www.openstudio.net">OpenStudio</a> software development kit and suite of applications.
+        <a href="https://openstudio.net/">OpenStudio</a> software development kit and suite of applications.
       </p>
       <p>DOE releases major updates to EnergyPlus twice annually. An updated version {{ latestRelease.version }} with bug fixes was released on {{ latestRelease.date }}.</p>
       <p>EnergyPlus is free, open-source, and cross-platform—it runs on the Windows, Mac OS X, and Linux operating systems.
         Its development is funded by the U.S. Department of Energy’s (DOE) Building Technologies Office (BTO). Along with OpenStudio,
         EnergyPlus is part of BTO’s <a href="https://www.energy.gov/eere/buildings/building-energy-modeling">building energy modeling program</a> portfolio.</p>
-      <p>For questions, comments, or modification requests to this website please contact <a href="mailto:amir.roth@ee.doe.gov">amir.roth@ee.doe.gov</a></p>
+      <p>For questions, comments, or other queries, please see the README on our <a href="https://github.com/NREL/EnergyPlus">GitHub Repository</a></p>
```

Comment:
Per Amir's request, I took his email off, and instead just point users to our repo, which will have the latest links for support, etc.